nab-starter/src/main/java/ru/hh/nab/starter/server/logging/StructuredRequestLogger.java

```diff
@@ -39,7 +44,14 @@ public void log(Request request, Response response) {
     long executionTime = currentTimeMillis() - request.getTimeStamp();
     context.put("time", executionTime);
     context.put("method", request.getMethod());
-    context.put("uri", request.getHttpURI().getPathQuery());
+    context.put(
+        "uri",
+        Optional
+            .ofNullable(request.getHttpURI())
+            .map(HttpURI::getPathQuery)
+            .or(() -> Optional.ofNullable(request.getMetaData()).map(MetaData.Request::getURI).map(HttpURI::getPathQuery))
```

Comment:
может
```suggestion
            .ofNullable(request.getHttpURI())
            .or(() -> Optional.ofNullable(request.getMetaData()).map(MetaData.Request::getURI))
            .map(HttpURI::getPathQuery)
```
т.е. вопрос, может ли `getPathQuery()` вернуть `null` на урле, где есть path

Reply:
да, так лучше, поправил